### DISCUSSION TITLE
Docker exec doesn't work on 1.7.0+

### DISCUSSION THREAD

**Discussion by fswijmans** [grafana/pyroscope#3469] [Q&A]
In docker desktop, I'm using the exec function, but since 1.7.0 it stopped working for me.

I can't find anything related on the release notes. Can someone verify this is a feature or bug?

**Comment by korniltsev** [comment]
https://github.com/grafana/pyroscope/commit/d2ba4ec7569593cf8b198efef5ecb5cc56f24cf4

What exactly are you trying to exec?

**Comment by fswijmans** [comment]
I'm trying to run a wget command inside the image, to verify it is running. `wget -q --spider http://localhost:4040` or something similar. This already worked before, but stopped working.

**Reply by korniltsev** [reply]
There is no wget in the docker image anymore. You can try using `profilecli`

This user has similar issues, it may be worth to track the issue https://github.com/grafana/pyroscope/pull/3468

**Comment by fswijmans** [comment]
Does the command `profilecli query series` exit with 1 if it's not ready/healthy?
